picard/releasegroup.py

```diff
@@ -77,25 +77,14 @@ def _parse_versions(self, document):
                 "id":      node['id'],
                 "year":    node['date'][:4] if "date" in node else "????",
                 "country": "+".join(countries) if countries
-                    else node['country'] if "country" in node
-                    else "??",
+                    else node.get('country', '') or "??",
```

Comment:
Why not just `node.get('country', '??')` (similarly for barcode and packaging)?

Reply:
There are times when node has a value for country, but it is set to None. It is to handle cases like that.

Reply:
The alternative would be to use something like node.get('country', '??') or '??'